includes/class-give-db.php

```diff
@@ -335,6 +335,33 @@ public function table_exists( $table ) {
 		return $wpdb->get_var( $wpdb->prepare( "SHOW TABLES LIKE '%s'", $table ) ) === $table;
 	}
 
+	/**
+	 * Checks whether column exists in a table or not.
+	 *
+	 * @param string $column_name Name of the Column in Database Table.
+	 *
+	 * @since 1.8.18
+	 *
+	 * @see https://gist.github.com/datafeedr/54e89e07f87232fb055121bb766743fe
+	 *
+	 * @return bool
+	 */
+	public function is_column_exists( $column_name ) {
```

Comment:
@mehul0810 I'd love to get this renamed to `does_column_exist( $column_name )` pre-1.8.18 release

Reply:
@mehul0810 I'll update it